internal/configs/configurator.go

```diff
@@ -30,6 +37,15 @@ const WildcardSecretName = "wildcard"
 // JWTKeyKey is the key of the data field of a Secret where the JWK must be stored.
 const JWTKeyKey = "jwk"
 
+// SPIFFE filenames and modes
+const (
+	spiffeCertFileName   = "spire_cert.pem"
```

Comment:
should the naming spiffe/spire be consistent? The variables are called like `spiffeCertFileName` but the filename cert is `spire_cert.pem`

Reply:
I see how this is confusing, the naming should be consistent. Since these certs are generated from the SVIDs, SPIFFE Verifiable Identity Documents, I think it makes name the certs `spiffe` instead of `spire`